keyboards/kodeboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,623 @@
+        KC_P4,   KC_P5,   KC_P6,   KC_PAST,
+        KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
+        KC_P0,   KC_PDOT, KC_PENT, KC_SPACE
+    ),
+    [_KL] = LAYOUT_k0depad(
+        KC_P7,   KC_P8,   KC_P9,   TO(_PL),
+        KC_P4,   KC_P5,   KC_P6,   KC_PAST,
+        KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
+        KC_P0,   KC_PDOT, KC_PENT, KC_SPACE
+    ),
+    [_PL] = LAYOUT_k0depad(
+        KC_P7,   KC_P8,   KC_P9,   TO(_AL),
+        KC_P4,   KC_P5,   KC_P6,   KC_PAST,
+        KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
+        KC_P0,   KC_PDOT, KC_PENT, KC_SPACE
+    ),
+    [_AL] = LAYOUT_k0depad(
+        KC_P7,   KC_P8,   KC_P9,   TO(_DL),
+        KC_P4,   KC_P5,   KC_P6,   KC_PAST,
+        KC_P1,   KC_P2,   KC_P3,   KC_PMNS,
+        KC_P0,   KC_PDOT, KC_PENT, KC_SPACE
+    ),
+};
+/*joystick_config_t joystick_axes[JOYSTICK_AXIS_COUNT] = {
+    JOYSTICK_AXIS_IN(GP27, 0, 512, 1023),
+    JOYSTICK_AXIS_IN(GP26, 0, 512, 1023),
+    
+}; */
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
+    [0] = {ENCODER_CCW_CW(KC_VOLD, KC_VOLU)  },
+    [1] = {ENCODER_CCW_CW(KC_A, KC_D)  },
+    [2] = {ENCODER_CCW_CW(RGB_SPD, RGB_SPI)  },
+    [3] = {ENCODER_CCW_CW(KC_RIGHT, KC_LEFT) },
+    [4] = {ENCODER_CCW_CW(KC_VOLD, KC_VOLU)  },
+    [5] = {ENCODER_CCW_CW(KC_A, KC_D)  },
+    [6] = {ENCODER_CCW_CW(RGB_SPD, RGB_SPI)  },
+    [7] = {ENCODER_CCW_CW(KC_RIGHT, KC_LEFT) },
+};
+#endif
+/*#ifdef OLED_ENABLE
+
+bool oled_task_user() {
+
+    oled_set_cursor(0, 32);
+    // Host Keyboard Layer Status
+    oled_write_P(PSTR("K0DEBOARD Ver:1.0.1 \nLAYER-"), false);
+    switch (get_highest_layer(layer_state)) {
+        case _DL:
+            oled_write_P(PSTR("1-SKETCHBOOK\n"), false);
+            break;
+        case _FL:
+            oled_write_P(PSTR("2-BLENDER\n"), false);
+            break;
+        case _CL:
+            oled_write_P(PSTR("3-IDE\n"), false);
+            break;
+        case _ML:
+            oled_write_P(PSTR("4-ASUS\n"), false);
+            break;
+        case _TL:
+            oled_write_P(PSTR("5-FIGMA\n"), false);
+            break; 
+        case _KL:
+            oled_write_P(PSTR("6-ADOBE-XD\n"), false);
+            break;
+        case _PL:
+            oled_write_P(PSTR("7-VS CODE\n"), false);
+            break;
+        case _AL:
+            oled_write_P(PSTR("8-GAME\n"), false);
+            break;        
+        default:
+            // Or use the write_ln shortcut over adding '\n' to the end of your string
+            oled_write_ln_P(PSTR("Undefined"), false);
+    }
+
+    // Host Keyboard LED Status
+    led_t led_state = host_keyboard_led_state();
+    oled_write_P(led_state.num_lock ? PSTR("[ N ]") : PSTR("[   ]"), false);
+    oled_write_P(led_state.caps_lock ? PSTR("[ CAPS ]") : PSTR("[      ]"), false);
+    oled_write_P(led_state.scroll_lock ? PSTR("[ SCRL ]") : PSTR("[      ]"), false);
+
+
+    
+    
+return false;
+}*/
+#ifdef OLED_ENABLE
```

Comment:
This code should be moved to the keyboard's c file, and use the kb versions of the functions.